docs/05-configuration.md

```diff
@@ -50,6 +50,7 @@ $ snowpack dev --no-bundle
   "scripts": { /* ... */ },
   "installOptions": { /* ... */ },
   "devOptions": { /* ... */ },
+  "buildOptions": { /* ... */ },
```

Comment:
+1, plenty of things in `devOptions` should really be moved to `buildOptions`. It would be a good follow-up to this PR to migrate the relevant ones over from `devOptions -> buildOptions`, if we could continue to support the legacy `devOptions`  usage in our `normalizeConfig()` function.